src/developer/web-api/tracker.md

```diff
@@ -2924,6 +2924,12 @@ Tracker objects, the requested organisation unit must always be within the user'
 user cannot request objects outside its search scope unless it's using the organisation unit mode
 ALL and has enough privileges to use that mode.
 
+When requesting tracked entities without specifying a program, the response will include only those tracked entities that satisfy one of the following criteria:
+- The tracked entity is enrolled in at least one program the user has data access to, and the user has access to the owner organisation unit.
+- The tracked entity is not enrolled in at least one program the user has data access to, and the user has access to the tracked entity registering organisation unit.
+
+In both scenarios, the validation process will consider the program access level, as detailed at the beginning of this section, and the sharing settings, as explained [here](#webapi_tracker_metadata_sharing).
```

Comment:
Can we merge this with 

> When requesting tracked entities without specifying a program, the response will include only those tracked entities that satisfy one of the following criteria:

to simplify things i.e. no reference back to the paragraph above.

How about

When requesting tracked entities without specifying a program, the response will include only tracked entities that satisfy [metadata sharing settings](#webapi_tracker_metadata_sharing) and one of the following criteria: